src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -436,7 +436,7 @@ public static void SupportNonNullableReferenceTypes(this SwaggerGenOptions swagg
             swaggerGenOptions.SchemaFilter<XmlCommentsSchemaFilter>(xmlDoc);
 
             if (includeControllerXmlComments)
-                swaggerGenOptions.DocumentFilter<XmlCommentsDocumentFilter>(xmlDoc);
+                swaggerGenOptions.DocumentFilter<XmlCommentsDocumentFilter>(swaggerGenOptions.SwaggerGeneratorOptions, xmlDoc);
```

Comment:
Looks like these parameters are the wrong way around - it concerns me that none of the tests have failed because of that 😓

Reply:
It appears that it actually worked with the arguments in the wrong order, but I went ahead and fixed it.

I also added a unit test to prove that it works, maybe it should be moved somewhere else, but I couldn't find a similar unit test class.